CONTRIBUTING.md

```diff
@@ -8,48 +8,48 @@ Welcome to [TheAlgorithms/C-Plus-Plus](https://github.com/TheAlgorithms/C-Plus-P
 
 ### Contributor
 
-We are very happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
+We are happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
 
 - You did your own work.
   - No plagiarism allowed. Any plagiarized work will not be merged.
 - Your work will be distributed under [MIT License](License) once your pull request has been merged.
-- You submitted work fulfils or mostly fulfils our styles and standards.
+- Your submitted work fulfills or mostly fulfills our styles and standards.
```

Comment:
```suggestion
- Please follow the repository coding guidelines (mentioned below)
```

Reply:
Also add one more point
- Please make sure that you do not provide any duplicate work (in a sense that implementation of data structure or algorithm is already present in repository)

Reply:
> Suggestion

What about `Please follow the repository guidelines and standards mentioned below`?